php/pantheon/checks/plugins.php

```diff
@@ -122,28 +132,23 @@ protected function getPluginVulnerability( $plugin_slug )
 
 
 	protected function getWpScanApiToken() {
```

Comment:
see comment above re: this method being consistent and cleaner about return type.